src/client/java/teammates/client/scripts/scalabilitytests/InstructorFeedbackResultsPageScalabilityTest.java

```diff
@@ -18,20 +33,196 @@
 
     private static final String DATA_FOLDER_PATH = "src/client/java/teammates/client/scripts/scalabilitytests/data/";
     private static final Logger log = Logger.getLogger();
+    private static DataBundle testDataMax;
 
     @Override
     protected void prepareTestData() {
         // the actual test data is refreshed before each test method
     }
 
     private void refreshTestData(int numStudents, int numQuestions) {
```

Comment:
This whole method (with all its helper methods) should be removed, as no modifications to the datastore should be done after the very first check. It is `testContentWithIncreasingLoad` that must be updated to load different pages (each corresponding to a different data set that already exists), instead of refreshing test data and loading the same page.